titus_isolate/allocate/constants.py

```diff
@@ -26,6 +26,7 @@
 TITUS_ISOLATE_CELL_HEADER = "X-Titus-Isolate-Cell"
 UNKNOWN_CELL = "unknown_cell"
 CELL = "cell"
+GRPC_CELL = "grpc_cell"
```

Comment:
Why is this needed?  Shouldn't a client land in a particular cell regardless of implementation?  I'd expect the `get_cell` function to determine which API it should address and return the correct cell.